src/Stratis.Bitcoin.Features.Consensus/Rules/ConsensusRule.cs

```diff
@@ -17,8 +17,9 @@ public abstract class ConsensusRule
         /// <summary>Instance logger.</summary>
         public ILogger Logger { get; set; }
 
+        //TODO: should this be IConsensusRules?
```

Comment:
I think it should. But in order to do that you need to make IConsensusRules interface to reflect every member of the actual ConsensusRules

@dangershony

Reply:
let's keep the TODO for now then

Reply:
I prefer to keep the todo for now as we still need to revisit how we measure download speed